bat/tests/09-no-delta-packs-over-format-bump/run.bats

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+# TODO: just call the auto-format-bump capability when available. Not using the old
+# auto-format-bump functionality because it will be changing very soon.
+
+@test "Attempt to create delta packs over format bump" {
+  mixer-init-stripped-down $CLRVER 10
+  sed -i 's/\(FORMAT\).*/\1 = "1"/' mixer.state
```

Comment:
Optional: to better future proof the regex, consider using `^\(FORMAT\) =.*` instead.